.github/workflows/apidoc.yaml

```diff
@@ -0,0 +1,84 @@
+name: Verify and Publish OpenAPI Specs
+
+on:
+  push:
+  pull_request:
+    branches: [ main ]
```

Comment:
Does this `branches: [ main ]` need to be moved under the `push` block?
Otherwise this workflow will be executed for every push and this `pull_request` block would be useless.

Reply:
on the upstream repo, the only pushes should be to `main`, and after merging a PR. I copied the declaration from the `verify.yaml`.